bundles/core/org.eclipse.smarthome.core/src/main/java/org/eclipse/smarthome/core/common/registry/AbstractRegistry.java

```diff
@@ -254,6 +261,13 @@ protected void addProvider(Provider<E> provider) {
             elementMap.put(provider, elements);
             for (E element : elementsOfProvider) {
                 try {
+                    K uid = element.getUID();
+                    if (uid != null && get(uid) != null) {
+                        logger.warn(String.format(
```

Comment:
Is there a special reason why you use `String.format` and not simply use parameterized logging?